src/main/java/org/broadinstitute/consent/http/db/DarCollectionDAO.java

```diff
@@ -121,25 +121,28 @@ List<DarCollection> findDARCollectionByCollectionIds(
   @RegisterBeanMapper(value = UserProperty.class, prefix = "up")
   @UseRowReducer(DarCollectionReducer.class)
   @SqlQuery(
-      "SELECT c.*, " +
-          User.QUERY_FIELDS_WITH_U_PREFIX + QUERY_FIELD_SEPARATOR +
-          Institution.QUERY_FIELDS_WITH_I_PREFIX + QUERY_FIELD_SEPARATOR +
-          UserProperty.QUERY_FIELDS_WITH_UP_PREFIX + QUERY_FIELD_SEPARATOR +
-          "dar.id AS dar_id, dar.reference_id AS dar_reference_id, dar.collection_id AS dar_collection_id, "
-          +
-          "dar.parent_id AS dar_parent_id, dar.draft AS dar_draft, dar.user_id AS dar_userId, " +
-          "dar.create_date AS dar_create_date, dar.sort_date AS dar_sort_date, dar.submission_date AS dar_submission_date, "
-          +
-          "dar.update_date AS dar_update_date, (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data, dd.dataset_id " +
-          "FROM dar_collection c " +
-          "INNER JOIN users u ON c.create_user_id = u.user_id " +
-          "LEFT JOIN user_property up ON u.user_id = up.user_id " +
-          "LEFT JOIN institution i ON i.institution_id = u.institution_id " +
-          "INNER JOIN data_access_request dar ON c.collection_id = dar.collection_id " +
-          "LEFT JOIN dar_dataset dd on dd.reference_id = dar.reference_id " +
-          "WHERE c.collection_id = (SELECT collection_id FROM data_access_request WHERE reference_id = :referenceId) "
-          +
-          "AND (LOWER(data->>'status')!='archived' OR data->>'status' IS NULL) ")
+      """
```

Comment:
It would've been nice to PR the text block change separately as it makes it hard to see the actual diff here.

Also note that changing to text block changes the text generated here. Originally it generated a string without line breaks, as a text block there are now line breaks. This shouldn't be a problem but I've seen cases where it can mess up logging, if SQL statements are logged without removing line breaks first.

Reply:
It would have.  Unfortunately what this effort shows is that we added semgrep and it's only looking at changed files so we're finding the problems as we go.  You'll see failed CI/CD checks for runs before I had to reformat the SQL.

I'd like to discuss the idea of a ticket to have a wholesale fix for this as just a SQL formatting PR since it's a pain to switch formatting of unrelated code.  Suspecting that ticket to be back burnered, I went ahead and only changed the files that were necessary for this PR.

Reply:
@pshapiro4broad - I created [https://broadworkbench.atlassian.net/browse/DT-1551](https://broadworkbench.atlassian.net/browse/DT-1551) for discussion in today's standup parking lot.